test/unit/util.test.js

```diff
@@ -60,6 +60,19 @@ describe('utility helpers', function () {
         });
     });
 
+    describe('type checkers', function () {
+        it('should validate integers', function () {
+            expect(util.isInt('123')).to.be.true;
+            expect(util.isInt('123.5')).to.be.false;
+        });
+
+        // @todo some issue with the function or my understanding of it's usage
+        it.skip('should validate floating point', function () {
+            expect(util.isFloat('123')).to.be.false;
```

Comment:
`123` is a valid float number. The implementation is:
```javascript
    isFloat: function (value) {
        return (value - parseFloat(value) + 1) >= 0;
    }
```

I think this function assumes `float` to be a superset of `int`. cc @codenirvana

Reply:
Yeah this is lifted straight from the parseCSV module and they have it like this too ... weird

Reply:
This is unrelated to this PR. So, let's leave it with @todo here for posterity